azure-pipelines.yml

```
@@ -0,0 +1,18 @@
+jobs:
+- job: macOS
+  pool:
+    vmImage: xcode9-macos10.13
+  steps:
+    - bash: |
+        export HOMEBREW_COLOR=1
+        export HOMEBREW_DEVELOPER=1
+        export HOMEBREW_NO_AUTO_UPDATE=1
+        brew update
+        # Mirror the repo as a tap.
+        HOMEBREW_TAP_DIR="/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask"
+        mkdir -p "${HOMEBREW_TAP_DIR}"
+        rsync -az --delete "${HOMEBREW_TAP_DIR}/"
+        export TRAVIS_BUILD_DIR="${HOMEBREW_TAP_DIR}"
+        builtin cd "${TRAVIS_BUILD_DIR}"
+        brew cask ci
+      displayName: Run brew cask-test-bot
```

Shouldn’t this start with `-` as well? As in:

```suggestion
      - displayName: Run brew cask-test-bot
```

Also, it’s my understanding this is pretty much just a message. So shouldn’t we change this for something more descriptive and that looks less like an outright command?

---

Finally, having it at the top might be clear visually.

---

> Shouldn’t this start with - as well?
[Not really](https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=vsts&tabs=schema#tabpanel_8ugYiRH0yv-6_schema) it works like this too.

> Finally, having it at the top might be clear visually.

Sure.